clang/test/SemaSYCL/loop_fusion.cpp

```diff
@@ -4,8 +4,8 @@
 
 [[intel::loop_fuse("foo")]] void func() {} // expected-error{{'loop_fuse' attribute requires an integer constant}}
 
-[[intel::loop_fuse(1048577)]] void func1() {}        // expected-error{{'loop_fuse' attribute requires integer constant between 0 and 1048576 inclusive}}
-[[intel::loop_fuse_independent(-1)]] void func2() {} // expected-error{{'loop_fuse_independent' attribute requires integer constant between 0 and 1048576 inclusive}}
+[[intel::loop_fuse(1048577)]] void func1() {}        // OK
```

Comment:
I am okay with the changes.   Just a question about the upper bound.   Is there an upper bound at all now?

Reply:
For these attributes, there isn't. As discussed in https://github.com/intel/llvm/issues/2911, there is no need for an upper bound since the backend doesn't care.